test/e2e/utils.go

```diff
@@ -170,31 +171,19 @@ func setDesiredStateWithPolicy(name string, desiredState nmstatev1alpha1.State)
 	setDesiredStateWithPolicyAndNodeSelector(name, desiredState, map[string]string{})
 }
 
-func updateDesiredStateAtNode(node string, desiredState nmstatev1alpha1.State) {
-	key := types.NamespacedName{Name: node}
-	state := nmstatev1alpha1.NodeNetworkState{}
-	Eventually(func() error {
-		err := framework.Global.Client.Get(context.TODO(), key, &state)
-		if err != nil {
-			return err
-		}
-		state.Spec.DesiredState = desiredState
-		return framework.Global.Client.Update(context.TODO(), &state)
-	}, ReadTimeout, ReadInterval).ShouldNot(HaveOccurred(), string(desiredState))
+func updateDesiredState(desiredState nmstatev1alpha1.State) {
+	setDesiredStateWithPolicy(DefaultPolicy, desiredState)
 }
 
-func updateDesiredState(desiredState nmstatev1alpha1.State) {
-	for _, node := range nodes {
-		updateDesiredStateAtNode(node, desiredState)
-	}
+func updateDesiredStateAtNode(node string, desiredState nmstatev1alpha1.State) {
+	nodeSelector := map[string]string{"kubernetes.io/hostname": node}
+	setDesiredStateWithPolicyAndNodeSelector(DefaultPolicy, desiredState, nodeSelector)
 }
 
 // TODO: After we implement policy delete (it will cleanUp desiredState) we have
 //       to remove this
 func resetDesiredStateForNodes() {
```

Comment:
Is this comment still valid?

Reply:
Nop, what we do is to ensure desiredState is cleanup but setting primary nic up

Reply:
AFAIK the comment was made when we though we would de-configure desiredState when it is removed. So the bridge would be rolled back. We decided that that won't be the case, so we will use resetDesiredStateForNodes forever. Thus we could drop it.